### PR TITLE
Create create_merchant method; needs refactor

### PR DESCRIPTION
**Pull request by jamogriff**
Functionality for creating Merchant  instances is complete. Refactor is needed to move it under `MerchantRepository` instead of `SalesEngine`

### CODE REVIEW

File: lib/sales_engine.rb [hunk 1]
```diff
@@ -28,7 +28,14 @@ def self.parse_csv(path)
...
   def merchants
-    MerchantRepository.new(parse_csv(@merchants))
+    MerchantRepository.new(create_merchants(parse_csv(@merchants)))
+  end
+
+  def create_merchants(parsed_data)
+     test = parsed_data.map do |merchant|
+      Merchant.new(merchant)
+    end
+     require 'pry'; binding.pry
```
[the remark this answers is not in the document]
Comment (jamogriff) [reply lib/sales_engine.rb:38]: Oopsie

File: lib/sales_engine.rb [hunk 2]
```diff
@@ -28,7 +28,14 @@ def self.parse_csv(path)
   end
 
   def merchants
-    MerchantRepository.new(parse_csv(@merchants))
+    MerchantRepository.new(create_merchants(parse_csv(@merchants)))
+  end
+
+  def create_merchants(parsed_data)
```
Comment (lcpulzone) [inline lib/sales_engine.rb:34]: Excited about this progress, thank you for implementing this!

**Review by lcpulzone — COMMENTED** [review]
Looks good, hopefully this didn't keep you up late at night!  Stoked to integrate this into Items.

**Review by aliyamerali — APPROVED** [review]
Looks good!

File: lib/sales_engine.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthewjholmes) [reply lib/sales_engine.rb:38]: No worries, let's just remove this before we move it.

**Review by matthewjholmes — APPROVED** [review]
Great work, Jamison

**Pull request merged**